i18n/zh-cn.toml

```diff
@@ -0,0 +1,26 @@
+[Search-placeholder]
+other = "搜索..."
+
+[Clear-History]
+other = "清理历史记录"
+
+[Attachments-label]
+other = "附件"
+
+[title-404]
+other = "错误"
+
+[message-404]
+other = "哎哟。 看起来这个页面不存在 ¯\\_(ツ)_/¯。"
+
+[Go-to-homepage]
+other = "转到主页"
+
+[Edit-this-page]
+other = "Edit this page"
+
+[Shortcuts-Title]
+other = "更多"
+
+[Expand-title]
+other = "Expand me..."
```

Comment:
No translation for this ?

Reply:
Chinese translation page, these two words will retain the English.

* `Edit this page`
* `Expand me...`

Reply:
Alright